python/sdk/prelude_sdk/controllers/jobs_controller.py

```diff
@@ -23,4 +29,8 @@ def job_status(self, job_id: str):
             headers=self.account.headers,
             timeout=30,
         )
-        return res.json()
+        job = res.json()
+        if self.account.resolve_enums:
+            self.resolve_enum(job, Control, "control")
```

Comment:
not gonna block on it, this is good, but would be cleaner to let resolve_enum take a list of tuples or dicts instead of calling resolve_enum multiple times

I was thinking of hiding this logic away in @pretty_print but now that i think about it, no real reason to not expose the logic to sdk users too. I might've still stuck it in the @verify_credentials decorator or something, but it's so minimal here that it's not really necessary